build/BranchInfo.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <Channel>release/15.5</Channel>
-    <BranchName>release/15.5</BranchName>
+    <Channel>release/2.0</Channel>
```

Comment:
Why is the channel release/2.0?

Reply:
Because this is going to be the latest CLI for .net core 2.0. Didn't we say that we would move our current 2.1 to the release/2.0 channel? So that people can acquire both the runtime and the SDK using a matching channel.